entries/jQuery.widget.xml

```diff
@@ -200,6 +209,106 @@
+				<argument name="extra" type="String" optional="true">
+					<desc>Additional classes to remove, required for layout or other reasons. Unlike the <code>keys</code> argument, these aren't associated with any properties of the <code>classes</code> option.</desc>
+				</argument>
+				<example>
+					<desc>Remove the <code>ui-progressbar</code> class from the widget's element (<code>this.element</code>). Will also remove any additional classes specified through the <code>classes</code> option for the given class.</desc>
+					<code><![CDATA[
+this._removeClass( "ui-progressbar" );
+]]></code>
+				</example>
+				<example>
+					<desc>Remove the <code>demo-popup-header</code> class from the specified element (here referencing <code>this.popup</code>). Will also remove any additional classes specified through the <code>classes</code> option for the given class. In addition, it will also remove the <code>ui-front</code> class.</desc>
+					<code><![CDATA[
+this._removeClass( this.popup, "demo-popup-header", "ui-front" );
+]]></code>
+				</example>
+				<example>
+					<desc>Remove the <code>ui-helper-hidden-accessible</code> class from the specified element. Uses <code>null</code> for the <code>keys</code> argument to skip it.</desc>
+					<code><![CDATA[
+this._removeClass( this.liveRegion, null, "ui-helper-hidden-accessible" );
+]]></code>
+				</example>
+			</method>
+			<method name="_toggleClass">
+				<desc>
+					Toggle classes of an element of the widget.
+					<p>The arguments are the same as for the <a href="#method-_addClass"><code>_addClass()</code></a> and <a href="#method-_removeClass"><code>_removeClass()</code></a> methods, except for the additional boolean argument that specifies to add or remove classes.</p>
+				</desc>
+				<argument name="element" type="jQuery" optional="true">
+					<desc>The element to toogle the classes on. Defaults to <code>this.element</code>.</desc>
+				</argument>
+				<argument name="keys" type="String">
+					<desc>
+						The classes to toogle, as a space-delimited list. If a property of the <a href="#option-classes"><code>classes</code> option</a> matches a key, the value will be toggled as well.
+						<p>When you only need the <code>extra</code> argument, you can skip this argument by specifying <code>null</code>.</p>
+					</desc>
+				</argument>
+				<argument name="extra" type="String" optional="true">
+					<desc>Additional classes to toggle, required for layout or other reasons. Unlike the <code>keys</code> argument, these aren't associated with any properties of the <code>classes</code> option. Just like <code>keys</code>, they will also be automatically removed when destroying the widget.</desc>
+				</argument>
+				<argument name="add" type="Boolean">
+					<desc>Indicates whether to add or remove the specified classes, where a boolean <code>true</code> indicates that classes should be added, a boolean <code>false</code> indicates that classes should be removed.</desc>
```

Comment:
Personally i would say this is a big enough deviation from the potentially "expected" behavior that it merits a warning